.github/workflows/KubernetesExternalClients.yaml

```diff
@@ -91,6 +91,7 @@ jobs:
           name: hz-hazelcast
         spec:
           clusterSize: 3
+          licenseKeySecretName: ${{ secrets.HAZELCAST_ENTERPRISE_KEY_V7 }}
```

Comment:
@yuce you can not use license-key as string directly, you need to create kubernets secret with it first then you will give secret name as `licenseKeySecretName` parameter:

```
    - name: Deploy Hazelcast cluster
      run: |-
        helm repo add hazelcast https://hazelcast-charts.s3.amazonaws.com/
        helm repo update
        helm install operator hazelcast/hazelcast-platform-operator --set installCRDs=true       
        kubectl create secret generic hazelcast-license-key --from-literal=license-key=<YOUR LICENSE KEY>
        cat <<EOF | kubectl apply -f -
        apiVersion: hazelcast.com/v1alpha1
        kind: Hazelcast
        metadata:
          name: hz-hazelcast
        spec:
          clusterSize: 3
          licenseKeySecretName: hazelcast-license-key
          exposeExternally:
            type: Smart
            discoveryServiceType: LoadBalancer
            memberAccess: LoadBalancer
        EOF
```

Reply:
Thanks,
Does the change in https://github.com/hazelcast/client-compatibility-suites/pull/125/commits/d1b7bd0ff467f5bc825fbdc52f58c52f6d24bb3e look OK?